build/repo.targets

```diff
@@ -40,18 +40,18 @@
     <Exec Command="dotnet run $(VerifierArgs)" WorkingDirectory="$(VerifierProjectDirectory)" />
   </Target>
 
-  <Target Name="RunSplitPackages" DependsOnTargets="RunResolvePackageDependencies">
+  <Target Name="RunSplitPackages" DependsOnTargets="RunResolvePackageDependencies;_FindDotNetPath">
     <ItemGroup>
       <SplitPackagePath Include="%(PackageDefinitions.ResolvedPath)" Condition="'%(PackageDefinitions.Name)'=='SplitPackages'" />
     </ItemGroup>
     <PropertyGroup>
       <ProductPackagesSource>$(ArtifactsDir)product-packages</ProductPackagesSource>
       <SplitPackagesCsv>$(RepositoryRoot)packages\packages.csv</SplitPackagesCsv>
-      <SplitPackagesBinary>@(SplitPackagePath)\SplitPackages.exe</SplitPackagesBinary>
```

Comment:
I haven't checked the SplitPackage change in yet, but just wanted to get this PR out first